### PR TITLE
renamed variables, some spacing changes, quotes

### PR DESCRIPTION
**Pull request by John151**
I renamed a few variables, took out some spacing and turned all double quotes " into single quotes '

### CODE REVIEW

**Comment by claraj** [comment]
@Nelwell can you review

**Comment by John151** [comment]
Finally passing tests, lmk if there's anything you'd like me to add or change, thanks!

File: convert_state.py [hunk 1]
```diff
@@ -1,60 +1,65 @@
+"""This code converts written state names to the 2 letter abbreviations, or the reverse"""
```
Comment (Nelwell) [inline convert_state.py]: """
This code converts written state names to the 2 letter abbreviations, or the reverse
"""

File: convert_state.py [hunk 2]
```diff
@@ -1,60 +1,65 @@
...
-
-
-import json  # import json for json processing
 def main():
-    with open('data.json','r') as file:
-        data = file.read()
-        data = json.loads(data)
+    """
+    reads in the data from state_data json file
+    """
```
Comment (Nelwell) [inline convert_state.py:9]: """
    Reads in the data from state_data json file
    """

File: convert_state.py [hunk 3]
```diff
@@ -1,60 +1,65 @@
...
+    with open('state_data.json','r') as file:
```
Comment (Nelwell) [inline convert_state.py:10]: with open('state_data.json', 'r') as file:

File: convert_state.py [hunk 4]
```diff
@@ -1,60 +1,65 @@
...
+        state_data = file.read()
+        state_abbr_json = json.loads(state_data)
 
-    print(data)
-    
     # Create a dictionary of states -> abbreviations, and a dictionary of abbreviations -> states 
-    stateAbbr = data #dictionary of state abbreviations keys and state name values 
-    stateAbbr2 = {}# dictionary of state name keys and state abbreviation values
+    # dictionary of state abbreviations keys and state name values
```
Comment (Nelwell) [inline convert_state.py]: unnecessary comment

File: convert_state.py [hunk 5]
```diff
@@ -1,60 +1,65 @@
...
+    state_abbr_to_longform = state_abbr_json  
+    state_longform_to_abbr = {} # dictionary of state name keys and state abbreviation values
```
Comment (Nelwell) [inline convert_state.py]: unnecessary comment

File: convert_state.py [hunk 6]
```diff
@@ -1,60 +1,65 @@
...
     # state key is the key in the state dictionary. state value is the value in the dictionary 
-    for statekey, statevalue in stateAbbr.items() :
-        stateAbbr2[statevalue] =  statekey
-
+    for statekey, statevalue in state_abbr_to_longform.items() :
```
Comment (Nelwell) [inline convert_state.py]: state_key and state_value for proper variable name convention

File: convert_state.py [hunk 7]
```diff
@@ -1,60 +1,65 @@
...
 
-
-    #loop
     while True:
-        print('1. Convert state to abbreviation')
-        print("2. Convery abreviasion to state")
-        print(' 3. quit')
-        choice =   input("Enter choice : ")
+        print('1. Convert state to abbreviation ')
+        print('2. Convery abreviasion to state ')
```
Comment (Nelwell) [inline convert_state.py]: typos

File: convert_state.py [hunk 8]
```diff
@@ -1,60 +1,65 @@
...
-    else:
-        print ( "The abbreviation for " +  userInput+" is " + result)
-def convert_abbreviation_to_state(dictionary):
-    userInput   = input("Enter abbrviation name").upper()
-    result = dictionary.get(userInput)
-    if result == None:
-        print(' abreviation not found')
-    else:
-        print("the state with the abbreviation   " + userInput + " is " + result)
+            print('try again ')
```
Comment (Nelwell) [inline convert_state.py:37]: Capitalize sentence.

File: convert_state.py [hunk 9]
```diff
@@ -1,60 +1,65 @@
...
+            
+def convert_state_to_abbreviation(dictionary):
```
Comment (Nelwell) [inline convert_state.py:39]: maybe better name for passed in dict.

File: convert_state.py [hunk 10]
```diff
@@ -1,60 +1,65 @@
...
+    """
+    user_input = input('Enter state name ').capitalize()
+    result = dictionary.get(user_input)
 
+    if result is None:
+        print('state not found ')
+    else:
+        print ('The abbreviation for ' +  user_input + ' is ' + result)
 
+def convert_abbreviation_to_state(dictionary):
```
Comment (Nelwell) [inline convert_state.py:51]: maybe better name for passed in dict.

File: convert_state.py (hunk 3 above)
Comment (John151) [reply convert_state.py:10]: I'm not sure what you mean here, I might be too sleepy, they look the same to me

File: convert_state.py [hunk 11]
```diff
@@ -1,60 +1,65 @@
...
-    result = dictionary.get(userInput)
-    if result == None:
-        print(' abreviation not found')
-    else:
-        print("the state with the abbreviation   " + userInput + " is " + result)
+            print('try again ')
+            
+def convert_state_to_abbreviation(dictionary):
+    """
+    function converts full written state to abbreviation, e.g. 'Minnesota' to 'MN'
```
Comment (Nelwell) [inline convert_state.py]: Capitalize docstring first word.

File: convert_state.py [hunk 12]
```diff
@@ -1,60 +1,65 @@
...
+    """
+    user_input = input('Enter state name ').capitalize()
```
Comment (Nelwell) [inline convert_state.py:43]: user_input = input('Enter state name: ').title()

File: convert_state.py [hunk 13]
```diff
@@ -1,60 +1,65 @@
...
+    result = dictionary.get(user_input)
 
+    if result is None:
+        print('state not found ')
```
Comment (Nelwell) [inline convert_state.py:47]: capitalize statement

File: convert_state.py (hunk 6 above)
Comment (John151) [reply convert_state.py]: very good catch, thanks

File: convert_state.py [hunk 14]
```diff
@@ -1,60 +1,65 @@
...
+    else:
+        print ('The abbreviation for ' +  user_input + ' is ' + result)
```
Comment (Nelwell) [inline convert_state.py:49]: could use f string here for formatting, looks cleaner.

File: convert_state.py [hunk 15]
```diff
@@ -1,60 +1,65 @@
...
 
+def convert_abbreviation_to_state(dictionary):
+    """
+    Converts state abbreviation to full name, e.g. 'WI' to 'Wisconsin'
+    """
+    user_input = input('Enter abbrviation name ').upper()
+    result = dictionary.get(user_input)
 
+    if result is None:
+        print('abreviation not found')
```
Comment (Nelwell) [inline convert_state.py:59]: capitalize statement.

File: convert_state.py [hunk 16]
```diff
@@ -1,60 +1,65 @@
...
+    else:
+        print('the state with the abbreviation ' + user_input + ' is ' + result)
```
Comment (Nelwell) [inline convert_state.py:61]: capitalize sentence and maybe ude f string for cleaner formatting.

File: convert_state.py [hunk 17]
```diff
@@ -1,60 +1,65 @@
...
 
 
 if __name__ == "__main__":
```
Comment (Nelwell) [inline convert_state.py:65]: unnecessary blank line.

File: convert_state.py [hunk 18]
```diff
@@ -1,60 +1,65 @@
...
```
Comment (Nelwell) [inline convert_state.py:64]: single quote here instead of double.

File: convert_state.py (hunk 3 above)
Comment (Nelwell) [reply convert_state.py:10]: oh i should have noted what I did lol.  I just added a space before the 'r', it's more proper.

**Review by Nelwell — COMMENTED** [review]
made comments per line.

**Pull request closed without merging**